src/state.rs

```diff
@@ -113,6 +117,125 @@ impl ToSerde<SerTaskState> for TaskState {
   }
 }
 
+#[derive(Clone,Copy,Debug,Deserialize,PartialEq,Serialize)]
+pub enum ColorDef {
+  Reset,
+  Rgb (u8, u8, u8)
+}
+
+impl ColorDef {
+  pub fn to_color(self) -> Box<dyn Color> {
+    match self {
+      ColorDef::Reset => Box::new(Reset),
+      ColorDef::Rgb(r, g, b) => Box::new(Rgb(r, g, b))
+    }
+  }
+}
+
+impl Default for ColorDef {
+  fn default() -> Self {
+      ColorDef::Reset
+  }
+}
```

Comment:
Hm, I guess you need that trait implementation because you want to have defaults for the individual colors below. I'd honestly try to avoid having a default color, as the notion is not intuitive to me. Can't we instead initialize each color with the appropriate default? Something along the lines of:
```rust
  #[serde(default = Rgb(...))]
  pub more_tasks_fg: ColorDef,
```
(or perhaps we need a separate function that doesn't take any arguments, not sure, see https://serde.rs/attr-default.html)

I think that would behave much more nicely if the user decides to only configure few of the colors. The way you implemented it, wouldn't the UI basically be all `Reset` if the user configured, say, only the `more_tasks_fg` color?

Reply:
You're right, that would result in strange behavior. I will see how I can handle this better.